spring-cloud-gateway-mvc/src/test/java/org/springframework/cloud/gateway/mvc/ProductionConfigurationTests.java

```diff
@@ -282,11 +282,12 @@ public void headers() throws Exception {
 						Map.class)
 				.getBody();
 		assertThat(headers).doesNotContainKey("foo").doesNotContainKey("hello")
-				.containsKeys("bar", "abc");
+				.containsKeys("bar", "abc", "authorization");
```

Comment:
This is not part of spring-cloud-gateway-server and should be removed from this enhancement since this module doesn't know anything about server properties.

Reply:
unders,i will remove modify about this.